backend/docker/prod/.env.example

```
@@ -0,0 +1,19 @@
+﻿# Docker Registry (e.g. ghcr.io/your-username)
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# Docker Registry (e.g. ghcr.io/your-username) key has incorrect delimiter

(IncorrectDelimiter)